src/Files.App/Views/Settings/GeneralPage.xaml

```diff
@@ -140,10 +140,10 @@
 								</Button.Flyout>
 								<StackPanel Orientation="Horizontal" Spacing="8">
 									<FontIcon
-										Margin="0,1,0,0"
-										FontSize="16"
+										VerticalAlignment="Center"
+										FontSize="14"
```

Comment:
General recommendation is **always** use sizes in a multiple of 4. Please use 16.

Reply:
@Jay-o-Way Thank you for the suggestion. The original reason for this rule doesn't apply anymore (it was originally for Windows Phone) but we still follow it in most cases as it's a good way to have a consistent design.

Reply:
I think it's still valid. I've seen it explicitly mentioned in Win 11 documentation. Reason is scaling factors, which also apply to desktops. Icons are also designed with a base size of 16 pixels. Using 14 will cause some blurring effects.